src/stilling/Stilling.js

```diff
@@ -50,6 +55,11 @@ class Stilling extends React.Component {
             document.title = this.props.stilling._source.title;
             this.hasSetTitle = true;
         }
+
+        if (!this.props.isFetchingStilling
```

Comment:
componentDidUpdate vil kjøres hver gang Stilling mottar nye props. Man trenger vel bare å sette meta-taggen en gang? Kunne man hatt en hasSetMetaRobotsTag=true|false i tillegg for å unngå at det gjøres flere ganger?